libs/s3/s3_versioning_common_test_lib.py

```diff
@@ -569,7 +575,10 @@ def update_versions_dict_dmo(versions_dict: dict, delete_result: list, is_versio
 
             versions_dict[obj]["version_history"].remove(ver)
             if ver == versions_dict[obj]["is_latest"]:
-                versions_dict[obj]["is_latest"] =  versions_dict[obj]["version_history"][-1]
+                if len(versions_dict[obj]["version_history"]) != 0:
```

Comment:
here you can just say for expected positive boolean if < 0
if len(versions_dict[obj]["version_history"]):

Reply:
I think it should be okay to leave it as it is as well